### PR TITLE
natron: init at 2.1.9

### PR DESCRIPTION
**Pull request by puffnfresh**
###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by rasendubi** [comment]
Failed to build on Gentoo+Nix:
```
rasen@ashmalko ~/nixpkgs> nix build natron
error: getting attributes of path ‘/home/rasen/nixpkgs/pkgs/applications/video/natron/config.pri’: No such file or directory
(use ‘--show-trace’ to show detailed location information)
```

**Comment by puffnfresh** [comment]
@rasendubi thanks I forgot to commit that file.

**Review by rasendubi — APPROVED** [review]
Superficially looks good to me. Builds fine on Gentoo+Nix.

File: pkgs/applications/video/natron/default.nix [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  src = fetchurl {
+    url = "https://github.com/MrKepzie/Natron/releases/download/${version}/Natron-${version}.tar.xz";
+    sha256 = "1wdc0zqriw2jhlrhzs6af3kagrv22cm086ffnbr1x43mgc9hfhjp";
+  };
+
+  buildInputs = [
+    pkgconfig qt4 boost expat cairo pythonPackages.pyside pythonPackages.pysideShiboken
+  ];
+
+  nativeBuildInputs = [ qmake4Hook python2Packages.wrapPython ];
```
Comment (FRidh) [inline pkgs/applications/video/natron/default.nix]: python2Packages or pythonPackages

Comment (Mic92) [inline pkgs/applications/video/natron/default.nix]: yes, not both.

Comment (puffnfresh) [inline pkgs/applications/video/natron/default.nix]: There's a `let pythonPackages = python2Packages` just above.

**Pull request merged**